requirements-local-forks.txt

```diff
@@ -3,4 +3,5 @@
 -e git+git://github.com/reanahub/reana-commons.git@master#egg=reana-commons
 -e git+git://github.com/cernanalysispreservation/invenio-oauthclient.git@master#egg=invenio-oauthclient
 -e git+git://github.com/cernanalysispreservation/invenio-jsonschemas.git@master#egg=invenio-jsonschemas
--e git+git://github.com/Lilykos/flask-celeryext.git@master#egg=flask-celeryext
\ No newline at end of file
+-e git+git://github.com/cernanalysispreservation/invenio-search.git@master#egg=invenio-search
```

Comment:
we should move those to the CAP user eventually

Reply:
ask @pamfilos to make a fork under analysispreservation organisation (i don't have rights)